code/modules/shuttle/emergency.dm

```diff
@@ -1,5 +1,6 @@
 #define TIME_LEFT (SSshuttle.emergency.timeLeft())
 #define ENGINES_START_TIME 100
+#define EARLY_AUTHORIZATION_DELAY_TIME 100
```

Comment:
Make a file called ``yogstation/code/modules/shuttle/emergency.dm`` and put this in there. Also include it in ``yogstation.dme``.

Reply:
Thanks, I had actually tried to modularize this a bit myself before seeing this and had put the define in this area:
`/code/__DEFINES/~yogs_defines/shuttles.dm`
Because I had noticed the shuttles.dm file existed in the standard TG defines as well, and the link you sent me explained I should place my defines in the ~yogs_defines folder. Any reason this doesn't go into the defines folder? Is it just to make it easier to find this way?